src/exporter.js

```diff
@@ -0,0 +1,174 @@
+const sha1 = require('sha1');
+
+export default class {
+  constructor(deckName) {
+    this.deckName = deckName;
+    this.db = this._getDb();
+    this.zip = this._getZip();
+    this.media = [];
+    this.topDeckId = rand();
+    this.topModelId = rand();
+    this.separator = SEPARATOR;
+    this.css = css;
+
+    return this
+      ._dbRun(this._getTemplate())
```

Comment:
That 3 methods are used only here and they are private, just copy all the code from them to the constructor